gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/KafkaAvroJobStatusMonitor.java

```diff
@@ -127,6 +127,8 @@ public org.apache.gobblin.configuration.State parseJobStatus(GobblinTrackingEven
       case TimingEvent.FlowTimings.FLOW_COMPILED:
         properties.put(JobStatusRetriever.EVENT_NAME_FIELD, ExecutionStatus.COMPILED.name());
         break;
+      case TimingEvent.LauncherTimings.WORK_UNITS_PREPARATION:
+      case TimingEvent.LauncherTimings.JOB_START:
```

Comment:
what is this launcher timings? It looks like it only sets a status but the job_start_time is when the job actually starts. Is the latter after we send request to azkaban and hear back it has started?